src/HLSMonitorService.ts

```diff
@@ -47,42 +45,115 @@ export class HLSMonitorService {
       exposeRoute: true,
     });
 
-    this.fastify.get("/status", async (request, reply) => {
-      if (!this.hlsMonitor) {
+    this.fastify.get("/monitor/:monitorId/status", async (request, reply) => {
+      if (!this.hlsMonitors.has(request.params.monitorId)) {
         reply.code(500).send({
           status: "error",
           message: "monitor not initialized",
         });
       }
-      const logs = await this.hlsMonitor.getErrors()
+    
+      const logs = await this.hlsMonitors.get(request.params.monitorId).getErrors()
       reply
         .code(200)
         .header('Content-Type', 'application/json; charset=utf-8')
         .send({ logs: logs });
     });
 
-    this.fastify.get("/clear-errors", async (request, reply) => {
-      if (!this.hlsMonitor) {
+    this.fastify.get("/monitor/:monitorId/clear-errors", async (request, reply) => {
```

Comment:
Perhaps we should have a DELETE on `/monitor/:monitorId/status` instead? I think that would be more restful

Reply:
I changed it to DELETE, not sure it is the correct method but lets go with it for now

Reply:
Yes, but change the resource to `/monitor/:monitorId/status` instead of `clear-errors`